components/director/internal/domain/application/service.go

```diff
@@ -193,6 +191,9 @@ func (s *service) Get(ctx context.Context, id string) (*model.Application, error
 
 	app, err := s.appRepo.GetByID(ctx, appTenant, id)
 	if err != nil {
+		if apperrors.IsNotFoundError(err) {
```

Comment:
On services level, please always return error. It is a GraphQL-specific case, when we shouldn't return an error - that's why the check should be done in resolver.